src/bun.js/api/bun/udp_socket.zig

```diff
@@ -153,9 +174,21 @@ pub const UDPSocketConfig = struct {
             }
         };
 
+        const flags: i32 = brk: {
+            if (try options.getTruthy(globalThis, "flags")) |value| {
```

Comment:
This should use validateInt from the validators so that it returns the correct error code for each scenario and so that it doesn't break if you do `13.0` instead of `13`
```suggestion
            if (try options.getTruthy(globalThis, "flags")) |value| {
              
```

Reply:
Doing this means I'd have to do some coercion in JS instead, because there are tests that are depending on the coercion.

Reply:
Actually, that's just for the port number. Should be fine with flags.